packages/ui/src/SmartResourcePropertyDisplay.tsx

```diff
@@ -0,0 +1,19 @@
+import { PropertyType } from '@medplum/core';
+import { Resource } from '@medplum/fhirtypes';
+import { evalFhirPath } from '@medplum/fhirpath';
+import React from 'react';
+import { ResourcePropertyDisplay } from '.';
+
+export interface SmartResourcePropertyDisplayProps {
+  resource: Resource;
+  path: string;
+}
+
+export function SmartResourcePropertyDisplay(props: SmartResourcePropertyDisplayProps): JSX.Element {
```

Comment:
Thoughts on the name?
* FhirPathDisplay
* FhirPathValueDisplay
* SmartFieldDisplay
* SmartPropertyDisplay